src/AspNetCoreMulti/Example/GraphQL/Queries.cs

```diff
@@ -0,0 +1,38 @@
+using GraphQL.Types;
+using System.Collections.Generic;
+
+namespace Example.GraphQL
+{
+    public class Queries
+    {
+        public class DogQuery : ObjectGraphType<object>
+        {
+            public DogQuery(IEnumerable<IDogOperation> dogOperations)
+            {
+                foreach(var dogOperation in dogOperations)
```

Comment:
```suggestion
                foreach (var dogOperation in dogOperations)
```

Reply:
I would be grateful if you copy the format ci here from the main repo. I have little time and I try to review a little everywhere where I promised. I still need to see two your PRs tonight.

Reply:
Sure; I'll make a new PR.